src/eos-kolibri-export

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+# Copyright © 2023 Endless OS Foundation, LLC
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or (at
+# your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.
+
+from argparse import ArgumentParser
+from gi.repository import Gio
+import logging
+import os
+import re
+import shlex
+import shutil
+import subprocess
+import time
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def system_server_proxy():
+    bus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
+    return Gio.DBusProxy.new_sync(
+        bus,
+        Gio.DBusProxyFlags.NONE,
+        None,  # interface info
+        'org.learningequality.Kolibri.Daemon',
+        '/org/learningequality/Kolibri/Daemon/Main',
+        'org.learningequality.Kolibri.Daemon',
+    )
```

Comment:
I'm not sure if it's worth the gymnastics of running this script inside the `org.learningequality.Kolibri` flatpak environment, but if we _did_ do that, we'd be able to use the `KolibriDaemonDBus` library. So, we'd get something like…

```python3
g_bus_type = KolibriDaemonDBus.get_default_bus_type()
return KolibriDaemonDBus.MainProxy(
    g_bus_type=g_bus_type,
    g_name=DAEMON_APPLICATION_ID,
    g_object_path=DAEMON_MAIN_OBJECT_PATH,
    g_interface_name=KolibriDaemonDBus.main_interface_info().name,
)

Reply:
Oh, that's pretty nice. There were a couple times I considered adding pieces that would run inside the sandbox but ultimately decided it would be easier for people to handle as a single script that could just be copied where needed.

Reply:
Having the script run on the host and be self-contained is useful for the offline case at hand, because then we don't need to also solve "reliably update kolibri flatpak from usb" to be able to solve "import new kolibri content from usb"